GoMain/src/main/main.go

```diff
@@ -75,6 +85,9 @@ func main() {
 		log.Fatalf("[%s] Orchestaration initalize fail : %s", logPrefix, err.Error())
 	}
 	sigmgr.Watch()
+	for {
+		time.Sleep(1000)
```

Comment:
Please see the merged PR https://github.com/lf-edge/edge-home-orchestration-go/pull/108/files. It is now deprecated. (Recommend you to sync up with the existing codes in the main-stream branch).

Reply:
Done